.github/workflows/manual-deploy-testnet-l1.yml

```diff
@@ -76,53 +68,52 @@ jobs:
         # DOCKER_BUILDKIT=1 will enable  the new docker build kit that allows us to use build only caches on RUN commands.
         # Tag the same image with 2 tags and push with the -a flag which pushes all images
         run: |
-          DOCKER_BUILDKIT=1 docker build -t ${{env.ETH2NETWORK_BUILD_TAG}} -t ${{env.L1_DOCKER_BUILD_TAG}} -f testnet/eth2network.Dockerfile .
-          docker push ${{env.L1_DOCKER_BUILD_TAG}}
-          docker push ${{env.ETH2NETWORK_BUILD_TAG}}
+          DOCKER_BUILDKIT=1 docker build -t ${{ vars.DOCKER_BUILD_TAG_ETH2NETWORK }} -f testnet/eth2network.Dockerfile .
+          docker push ${{ vars.DOCKER_BUILD_TAG_ETH2NETWORK }}
 
       # This will fail some deletions due to resource dependencies ( ie. you must first delete the vm before deleting the disk)
       - name: 'Delete deployed VMs'
         uses: azure/CLI@v1
         with:
           inlineScript: |
-            $(az resource list --tag ${{env.RESOURCE_TAG_NAME}}=true --query '[]."id"' -o tsv | xargs -n1 az resource delete --verbose -g Testnet --ids) || true
+            $(az resource list --tag ${{ vars.AZURE_RESOURCE_TAG_L1 }}=true --query '[]."id"' -o tsv | xargs -n1 az resource delete --verbose -g Testnet --ids) || true
```

Comment:
nit: `AZURE_RESOURCE_TAG_L1` -> `AZURE_DEPLOY_GROUP_L1`

nit: `AZURE_RESOURCE_TAG_L2` -> `AZURE_DEPLOY_GROUP_L2`

Went back and forth a few times because of the similarity with the `AZURE_RESOURCE_NAME_L1`, hopefully you agree this suggestion makes it clearer. ( Or something along these lines)

Reply:
Yeah nice I think that makes the intention clearer.